doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1450,6 +1418,54 @@ Definitions of recursive objects in co-inductive types
       As in the :cmd:`Fixpoint` command, it is possible to introduce a block of
       mutually dependent methods.
 
+.. _Computation:
+
+Computation
+-----------
+
+.. insertprodn reduce pattern_occ
+
+.. prodn::
+   reduce ::= Eval @red_expr in
+   red_expr ::= red
+   | hnf
+   | simpl {? @delta_flag } {? @ref_or_pattern_occ }
+   | cbv {? @strategy_flag }
+   | cbn {? @strategy_flag }
+   | lazy {? @strategy_flag }
+   | compute {? @delta_flag }
+   | vm_compute {? @ref_or_pattern_occ }
+   | native_compute {? @ref_or_pattern_occ }
+   | unfold {+, @unfold_occ }
+   | fold {+ @term1_extended }
+   | pattern {+, @pattern_occ }
+   | @ident
+   delta_flag ::= {? - } [ {+ @smart_global } ]
+   smart_global ::= @qualid
+   | @by_notation
+   by_notation ::= @string {? % @ident }
+   strategy_flag ::= {+ @red_flags }
+   | @delta_flag
+   red_flags ::= beta
+   | iota
+   | match
+   | fix
+   | cofix
+   | zeta
+   | delta {? @delta_flag }
+   ref_or_pattern_occ ::= @smart_global {? at @occs_nums }
+   | @term1_extended {? at @occs_nums }
+   occs_nums ::= {+ @num_or_var }
+   | - @num_or_var {* @int_or_var }
+   num_or_var ::= @num
+   | @ident
+   int_or_var ::= @int
+   | @ident
+   unfold_occ ::= @smart_global {? at @occs_nums }
+   pattern_occ ::= @term1_extended {? at @occs_nums }
+
+Details TBA.
```

Comment:
Instead, you can just put a link to the "Conversion rules" section of the CIC chapter and a `.. todo::`.